ASUWebPlatforms/webspark-ci
# WS2 2002: Fix list style issues in Renovation

### Description
There were several issues with lists that I have addressed with this PR. Please refer to [the ticket](https://asudev.jira.com/browse/WS2-2002) to see some of the major issues that it covers.

QA Test steps:
- Navigate to /text-content-lists. Notice that I have added a "Block version" at the bottom, which will allow for testing the lists in the OffCanvas dialog in Layout Builder, as well as in the "Edit" tab.
- Notice that the icon list no longer has a hanging indent on the second line.
- Click on the "Edit" tab. When the page loads, scroll down and notice the same thing.
- Hover over one of the icons. A gold box should appear around it.
- Click the gold box. It should turn blue. This means you can edit the icon. So, in the toolbar, click on the flag (Fontawesome) button. This will open a modal window where you can choose a new icon.
- Enter "cube" in the search box and then click on the suggestion when it loads, and then click "Insert Icon"
- Notice that the new cube icon has replaced the rocket one without messing up the layout. 
- Scroll down and click "Save." Notice that the cube icon displays correctly in the "View" version.
- Click on the "Layout" tab
- Scroll down to the block named "Block version" and click on the pencil icon and select "Configure."
- When the OffCanvas dialog loads, notice that the font is Arial, like in the View version.
- Scroll down and note that the icons appear correctly.
- Highlight all of the items in the icon list. Then click the "List properties" button in the toolbar. Select "Default" and click the green checkmark.
- Notice how the list becomes a bullet list. Also notice that if you hover over the icons the gold box may or may not surround the icon.
- Click into the text, just before the initial letter, and then press the Backspace (or Delete) button on your computer (depending on if you're using Windows or a Mac). This should delete the icon and the gold/blue box. You may need to press the key twice.
- Do this for each list item, so that it becomes a normal bullet list.
- Click the "update" button, and then click "Save layout."
- Notice that the list looks correct as a normal bullet list. Now go back into Layout builder and load up the WYSIWYG for the "Block version."
- Locate the bullet list that used to be the icon list.
- Click into the first item at the beginning of the line.
- Click the flag (FontAwesome) button in the toolbar and enter "rocket" into the field, select the correct result and then click "Insert icon"
- Do the same for each line, adding "bus," "car," and "bicycle."
- You should see a bullet list with icons at the beginning of each text line. Hover over each icon and it should properly be surrounded by a gold box.
- Click on the List Properties button in the toolbar and select "Icon list" and then click the green checkmark.
- The bullets should get removed and the icons should replace them as the new "bullets."
- Repeat the previous action of clicking on the gold box and replacing an icon. 
- Click on the "List properties" button in the toolbar. Select "Icon List Maroon" and click the green checkmark. All of the icons should have turned into maroon icons.
- Click on the gold box of one of the icons and replace it. The new icon should be maroon.
- Do the same with "Icon List Gray 7" and "Icon List Gray 7 Gold"
- If the colors change as expected, scroll down and click "Update."
-  When the OffCanvas dialog closes, click "Save layout" and note that the icon list looks the same in the "View" version.
- Load the page in Safari and check the lists in CKEditor (both in Edit and Layout Builder) to make sure that double bullets and double numbers aren't showing. 
- If all of these steps work as described, this passes QA.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-2002)

### Checklist


### Verified in browsers 


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
